Sources/IBLinterKit/UseBaseClassConfig.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BaseClassConfig.swift
+//  AEXML
+//
+//  Created by masamichi on 2019/03/11.
+//
+
+import Foundation
+
+public struct UseBaseClassConfig: Codable {
+    public let elementClass: String
+    public let baseClasses: [String]
+
+    enum CodingKeys: String, CodingKey {
+        case elementClass = "element_class"
+        case baseClasses = "base_classes"
+    }
+
+    public static let `default` = UseBaseClassConfig.init()
+
+    private init() {
+        elementClass = ""
+        baseClasses = []
+    }
+
+    init(elementClass: String, baseClasses: [String]) {
+        self.elementClass = elementClass
+        self.baseClasses = baseClasses
+    }
+
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        elementClass = try container.decodeIfPresent(Optional<String>.self, forKey: .elementClass).flatMap { $0 } ?? ""
```

Comment:
If `elementClass` is nil or not present, this should throw a decoding error.
But in the case of `baseClasses`, it's OK because empty array in YAML is expressed in the same way of `nil`

```yaml
elementClass: # This is invalid. Shouldn't be nil
baseClasses: # It's OK. This means empty array
```

Reply:
I'll fix to throw an error!